### ISSUE TITLE
Cannot retrieve template by id for builtin nodes

### ISSUE THREAD

**Issue by davidban77** [davidban77/gns3fy#7]
There is a bug for the devices that are builtin in GNS3 when trying to retrieve their template information by ID.

```
In [5]: for t in server.get_templates():
   ...:     if t["builtin"] is True:
   ...:         print(f"Name: {t['name']} -- {t['template_id']}")
   ...:
Name: Cloud -- 39e257dc-8412-3174-b6b3-0ee3ed6a43e9
Name: NAT -- df8f4ea9-33b7-3e96-86a2-c39bc9bb649c
Name: VPCS -- 19021f99-e36f-394d-b4a1-8aaa902ab9cc
Name: Ethernet switch -- 1966b864-93e7-32d5-965f-001384eec461
Name: Ethernet hub -- b4503ea9-d6b6-3695-9fe4-1db3b39290b0
Name: Frame Relay switch -- dd0f6f3a-ba58-3249-81cb-a1dd88407a47
Name: ATM switch -- aaa764e2-b383-300f-8a0e-3493bbfdb7d2

In [6]: server.get_template_by_id('39e257dc-8412-3174-b6b3-0ee3ed6a43e9')
Out[6]:
{'message': "'compute_id' is a required property\n\nFailed validating 'required' in schema:\n    {'$schema': 'http://json-schema.org/draft-04/schema#',\n     'description': 'A template object',\n     'properties': {'builtin': {'description': 'Template is builtin',\n                                'type': 'boolean'},\n                    'category': {'anyOf': [{'type': 'integer'},\n                                           {'enum': ['router',\n                                                     'switch',\n                                                     'guest',\n                                                     'firewall']}],\n                                 'description': 'Template category'},\n                    'compute_id': {'description': 'Compute identifier',\n                                   'type': 'string'},\n                    'default_name_format': {'description': 'Default name '\n                                                           'format',\n                                            'minLength': 1,\n                                            'type': 'string'},\n                    'name': {'description': 'Template name',\n                             'minLength': 1,\n                             'type': 'string'},\n                    'symbol': {'description': 'Symbol of the template',\n                               'minLength': 1,\n                               'type': 'string'},\n                    'template_id': {'description': 'Template UUID',\n                                    'maxLength': 36,\n                                    'minLength': 36,\n                                    'pattern': '^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$',\n                                    'type': 'string'},\n                    'template_type': {'description': 'Type of node',\n                                      'enum': ['cloud',\n                                               'ethernet_hub',\n                                               'ethernet_switch',\n                                               'docker',\n                                               'dynamips',\n                                               'vpcs',\n                                               'traceng',\n                                               'virtualbox',\n                                               'vmware',\n                                               'iou',\n                                               'qemu']}},\n     'required': ['name',\n                  'template_type',\n                  'template_id',\n                  'category',\n                  'compute_id',\n                  'default_name_format',\n                  'symbol',\n                  'builtin'],\n     'type': 'object'}\n\nOn instance:\n    {'builtin': True,\n     'category': 'guest',\n     'name': 'Cloud',\n     'symbol': ':/symbols/cloud.svg',\n     'template_id': '39e257dc-8412-3174-b6b3-0ee3ed6a43e9',\n     'template_type': 'cloud'}",
 'method': 'GET',
 'path': '/v2/templates/{template_id}',
 'request': {},
 'status': 400}
```

You can see that it requires `compute_id` for its schema validation. And by its [documentation](http://api.gns3.net/en/2.2/api/v2/controller/template/templatestemplateid.html#get-v2-templates-template-id) the `template_id` should be enough.

**NOTE:** This does not happen for devices added into GNS3 by the user, only builtin ones as far I have tested

**Comment by davidban77** [comment]
https://github.com/GNS3/gns3-server/issues/1638

**Comment by davidban77** [comment]
Fixed, will wait to test and close on next release

**Comment by davidban77** [comment]
Looks like is targeted for 2.2.0 release

**Comment by davidban77** [comment]
Fixed on 2.2.1
